src/parser/TerraformListener.js

```diff
@@ -226,40 +226,79 @@ class TerraformListener extends antlr4.tree.ParseTreeListener {
         this.currentField.name,
       );
     }
-
     this.currentField = null;
   }
 
   // Enter a parse tree produced by terraformParser#complexField.
-  enterComplexField(ctx) {
+  enterComplexField() {
```

Comment:
I'm not sure, but complexe field are remove in grammar, so you can delete related functions

Reply:
Done